### DISCUSSION TITLE
Issue when using starter template from local repository

### DISCUSSION THREAD

**Discussion by Borka1985** [quasarframework/quasar#12343] [General - Components / Directives / etc]
Hello. Created a custom
quasar-starter-kit, placed in the local gitlab repository, but I can't run quasar create --kit gitlab:https://gitlab.company.ru/templates/quasar-starter-kit.git or
quasar create --kit https://gitlab.company.ru/templates/quasar-starter-kit. I would be grateful for any help in solving the problem.

**Comment by MilosPaunovic** [comment]
Have you followed this instructions?
[https://quasar.dev/quasar-cli/commands-list#create](https://quasar.dev/quasar-cli/commands-list#create)

**Comment by Borka1985** [comment]
Yes, of course. We are using a local gitlab repository. Write how to correctly specify the url of the repository?

**Comment by MilosPaunovic** [comment]
You can use a starter kit stored into any publicly accessible Git repository by providing a reference which follows this schema:
```
GitHub - github:owner/name or simply owner/name
GitLab - gitlab:owner/name
Bitbucket - bitbucket:owner/name
```
`master` branch will be checked out by default, but you can specify the one you prefer via `--branch <branch name>` (eg. `quasar create --kit owner/name --branch my-branch`).

**Comment by Borka1985** [comment]
Thanks, but what about the local repository?
This is the correct command:
quasar create --kit gitlab:https://gitlab.company.ru/templates/quasar-starter-kit ?

**Comment by MilosPaunovic** [comment]
You can use a starter kit stored on your machine by providing a local path to a folder (eg. `quasar create --kit ./my-custom-starter-kit`).

**Comment by Borka1985** [comment]
Thanks, but I don't want to use a local folder. You need to use a local gitlab server. It's impossible?

**Comment by MilosPaunovic** [comment]
The repository you want to use needs to be publicly accessible.

**Comment by Borka1985** [comment]
Yes, the repository is public. But the gitlab server is deployed inside the enterprise. Can you send an example link to access the local gitlab server (self-managed GitLab)?
